benchmarks/gpu_pyfixest_errors.ipynb

```diff
@@ -0,0 +1,1478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        dum_all[:, 0] = np.random.choice(nb_dum[0], n, replace=True)
</pre><p>You could use a rng (<a href="https://numpy.org/doc/stable/reference/random/generator.html" target="_blank">https://numpy.org/doc/stable/reference/random/generator.html</a>) to make the examples reproducible </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>